bcipy/helpers/copy_phrase_wrapper.py

```diff
@@ -97,6 +97,10 @@ def __init__(self,
                  stim_length: int = 10,
                  stim_order: StimuliOrder = StimuliOrder.RANDOM):
 
+        if not lmodel:
+            assert is_txt_stim, "Language model is required."
```

Comment:
Here it seems like `UniformLanguageModel` will only work with text, but inside that class it seems like it can handle either text or img. (Maybe I'm misreading...)

Reply:
This is probably a bit confusing. The `UniformLanguageModel` can support both text or images. If the symbol set is text we know how to construct the alphabet, but if it's images we need more information that's not available at this point (the path to the images, etc.). It might be simpler to just always require a language model.